internal/provider/engine.go

```diff
@@ -153,41 +166,51 @@ var expressionsAttribute = schema.SetNestedAttribute{
 	NestedObject: schema.NestedAttributeObject{
 		Attributes: map[string]schema.Attribute{
 			"id": schema.StringAttribute{
-				Computed: true,
+				MarkdownDescription: apischema.Docstring("ExpressionV2ResponseBody", "id"),
+				Computed:            true,
 			},
 			"label": schema.StringAttribute{
-				Required: true,
+				MarkdownDescription: apischema.Docstring("ExpressionV2ResponseBody", "label"),
+				Required:            true,
 			},
 			"reference": schema.StringAttribute{
-				Required: true,
+				MarkdownDescription: apischema.Docstring("ExpressionV2ResponseBody", "reference"),
+				Required:            true,
 			},
 			"root_reference": schema.StringAttribute{
-				Required: true,
+				MarkdownDescription: apischema.Docstring("ExpressionV2ResponseBody", "root_reference"),
+				Required:            true,
 			},
 			"else_branch": schema.SingleNestedAttribute{
-				Optional: true,
+				MarkdownDescription: "The else branch to resort to if all operations fail",
+				Optional:            true,
 				Attributes: map[string]schema.Attribute{
 					"result": schema.SingleNestedAttribute{
-						Required:   true,
-						Attributes: paramBindingAttributes,
+						MarkdownDescription: "The result assumed if the else branch is reached",
+						Required:            true,
+						Attributes:          paramBindingAttributes,
 					},
 				},
 			},
 			"operations": schema.ListNestedAttribute{
-				Required: true,
+				MarkdownDescription: "The operations to execute in sequence for this expression",
```

Comment:
Some attributes in the openapi spec reference another type, and so I can't seem to find a description .. so I'm writing my own on occasion

I could check the design file